backend/authentication/models.py

```diff
@@ -0,0 +1,15 @@
+import os
+from django.dispatch import receiver
+from django.urls import reverse
+from django_rest_passwordreset.signals import reset_password_token_created
+from django.core.mail import send_mail
+from rest_framework.authtoken.models import Token
+
+
+@receiver(reset_password_token_created)
+def password_reset_token_created(reset_password_token: Token):
+    email_plaintext_message = "{}?token={}".format(
+        reverse("password_reset:reset-password-request"),
+        reset_password_token.key)
+    send_mail("Password Reset for Wine Card", email_plaintext_message,
+              os.getenv("SENDGRID_EMAIL").__str__(), [reset_password_token.user.email])
```

Comment:
`os.getenv` returns strings by default. And, anyway, it's better to write `str(object)` rather than when to use the object's magic method `object.__str__()`

```suggestion
              os.getenv("SENDGRID_EMAIL"), [reset_password_token.user.email])
```